### PR TITLE
[CP] For katello 3.13.4 release

### PR DESCRIPTION
**Pull request by jjeffers**
Bumps version to 3.13.4, cherry-picks for release, and an update to the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# 3.13.4 Baltic Porter (2020-01-13)
+
+## Features
+
+## Bug Fixes
+
+### Subscriptions
+ * custom products can't be consumed by hosts ([#28714](https://projects.theforeman.org/issues/28714), [9c28bc97](https://github.com/Katello/katello.git/commit/9c28bc97a553977269f2eeb5feed79370580a833), [eb233a31](https://github.com/Katello/katello.git/commit/eb233a310083413045bf63c634865ad4f3adf076))
```
Comment (jturel) [inline CHANGELOG.md]: The issue has some closed (unmerged) PRs associated - can you remove then and regenerate the changelog?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# 3.13.4 Baltic Porter (2020-01-13)
+
+## Features
```
Comment (jturel) [inline CHANGELOG.md]: empty features heading :(

File: CHANGELOG.md (hunk 1 above)
Comment (jturel) [reply CHANGELOG.md]: Needs another regenerate - the CP PR shouldn't be linked to the issue - I removed it for ya

**Comment by mmoll** [comment]
@jjeffers GH-8514 should fix the angular-ui test errors

File: CHANGELOG.md (hunk 1 above)
Comment (jjeffers) [reply CHANGELOG.md]: I may not understand this completely, but both of those commits are related. The first is a PR to address the original redmine issue, while the second commit it @jlsherrill 's PR to correct a VCR tape related to the same redmine issue.

When generating the CP from tool_belt, both commit SHA are indicated with the same description. Both commits can be picked without conflict, etc. Are you suggesting we drop one of those (the VCR update)? Let me know if I understand what you meant.

Comment (jturel) [reply CHANGELOG.md]: My bad, I was misinterpreting what I was reading here. Changelog looks good!

**Comment by jjeffers** [comment]
[test katello]

**Pull request merged**